app/views/casework/crime_applications/_case_details.html.erb

```diff
@@ -66,77 +66,74 @@
         <%= case_details.next_court_hearing_at %>
       </dd>
     </div>
-  </dl>
-<% end %>
-
-<% if FeatureFlags.means_journey.enabled? %>
-  <dl class="govuk-summary-list govuk-!-margin-bottom-9">
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:has_case_concluded) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= t(case_details.has_case_concluded, scope: 'values') || t('values.not_asked') %>
-      </dd>
-    </div>
-
-    <% if case_details.case_concluded? %>
+    <% if FeatureFlags.means_journey.enabled? %>
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          <%= label_text(:date_case_concluded) %>
+          <%= label_text(:has_case_concluded) %>
         </dt>
         <dd class="govuk-summary-list__value">
-          <%= l(case_details.date_case_concluded, format: :compact) %>
+          <%= t(case_details.has_case_concluded, scope: 'values') || t('values.not_asked') %>
```

Comment:
I think we're not going with this approach anymore? based on what fleur said here: https://mojdt.slack.com/archives/C02AFLHGK9S/p1706792275399049